src/Enecuum/Core/Runtime.hs

```diff
@@ -2,30 +2,64 @@ module Enecuum.Core.Runtime where
 
 import Enecuum.Prelude
 
-import qualified Enecuum.Core.Types as T
+import qualified Data.Map                          as Map
+import qualified Data.ByteString.Base64            as Base64
+import qualified Crypto.Hash.SHA256                as SHA
+
+import qualified Enecuum.Core.Types                as D
 import qualified Enecuum.Core.Logger.Impl.HsLogger as Impl
 
-data LoggerRuntime = LoggerRuntime
+newtype LoggerRuntime = LoggerRuntime
     { _hsLoggerHandle :: Maybe Impl.HsLoggerHandle
     }
 
 data CoreRuntime = CoreRuntime
     { _loggerRuntime :: LoggerRuntime
+    , _stateRuntime  :: StateRuntime
+    }
+
+newtype VarNumber = VarNumber Int
+
+data VarHandle = VarHandle D.VarId (TVar Any)
+
+instance D.StringHashable VarNumber where
+  toHash (VarNumber n) = D.StringHash . Base64.encode . SHA.hash $ show ("VarNumber " +|| n ||+ "" :: String)
```

Comment:
May be we need to unificate hashing function toHash
and use it here. 
And change it from
toHash = StringHash . Base64.encode . SHA.hash . S.encode
to
toHash = StringHash . Base64.encode . SHA.hash